test/k8sT/Policies.go

```diff
@@ -543,7 +543,7 @@ var _ = Describe("K8sPolicyTest", func() {
 			for _, pod := range []string{appPods[helpers.App2], appPods[helpers.App3]} {
 				res := kubectl.ExecPodCmd(
 					namespaceForTest, pod,
-					helpers.CurlFail("http://1.1.1.1/"))
+					helpers.CurlFail("--retry 5 http://1.1.1.1/"))
 				res.ExpectFail("Egress connectivity should be denied for pod %q", pod)
```

Comment:
Do the retries make sense for this one and all others below? The request is expected to fail so it's just going to take longer most of the time.

Reply:
I thought about this but then figured if the success test is not reliable then the fail case would provide false results. I'm OK to drop it though.

Reply:
Fixed